website/content/docs/operations/metrics-reference.mdx

```diff
@@ -93,13 +93,16 @@ be assumed to be 10 seconds when retrieving metrics using the above described
 | `nomad.runtime.alloc_bytes`                  | Memory utilization                                                                                                                                                                                                | # of bytes                     | Gauge   |
 | `nomad.runtime.heap_objects`                 | Number of objects on the heap. General memory pressure indicator                                                                                                                                                  | # of heap objects              | Gauge   |
 | `nomad.runtime.num_goroutines`               | Number of goroutines and general load pressure indicator                                                                                                                                                          | # of goroutines                | Gauge   |
 | `nomad.nomad.broker.total_pending`           | Evaluations that are pending until an existing evaluation for the same job completes                                                                                                                              | # of evaluations               | Gauge   |
 | `nomad.nomad.broker.total_ready`             | Number of evaluations ready to be processed                                                                                                                                                                       | # of evaluations               | Gauge   |
 | `nomad.nomad.broker.total_unacked`           | Evaluations dispatched for processing but incomplete                                                                                                                                                              | # of evaluations               | Gauge   |
+| `nomad.nomad.broker.wait_time`               | Time elapsed while the evaluation was ready to be processed and waiting to be dequeued                                                                                                                            | ns / Evaluation Wait           | Timer   |
+| `nomad.nomad.broker.process_time`            | Time elapsed while the evaluation was dequeued and finished processing                                                                                                                                            | ns / Evaluation Process        | Timer   |
+| `nomad.nomad.broker.response_time`           | Time elapsed from when the evaluation was last enqueued and finished processing                                                                                                                                   | ns / Evaluation Response       | Timer   |
```

Comment:
It may be worth expanding https://developer.hashicorp.com/nomad/docs/operations/monitoring-nomad#scheduling to mention how to use these metrics as well.

I think tracking `response_time` is going to be something important to track, and then `wait_time` and `process_time` can help nail down the problem. An increase in `wait_time` probably means too much load. An increase in `process_time` points more towards a server performance issue (CPU, disk, network etc.)

We likely won't know exactly what to write until we advance in our explorations, but good to keep in mind.